roles/confluent.variables/vars/main.yml

```
@@ -555,10 +617,28 @@ ksql_properties:
       websocket.servlet.initializor.classes: io.confluent.common.security.jetty.initializer.InstallBearerOrBasicSecurityHandler
       oauth.jwt.public.key.path: "{{rbac_enabled_public_pem_path}}"
       public.key.path: "{{rbac_enabled_public_pem_path}}"
-      confluent.metadata.bootstrap.server.urls: "{{mds_urls}}"
+      confluent.metadata.bootstrap.server.urls: "{{mds_bootstrap_server_urls}}"
       confluent.metadata.public.key.path: "{{rbac_enabled_public_pem_path}}"
       confluent.metadata.basic.auth.user.info: "{{ ksql_ldap_user | default('ksql') }}:{{ ksql_ldap_password | default('pass') }}"
       confluent.metadata.http.auth.credentials.provider: BASIC
+  rbac_external_client:
+    enabled: "{{rbac_enabled and external_mds_enabled and mds_broker_listener.ssl_enabled }}"
```

```suggestion
    enabled: "{{rbac_enabled and external_mds_enabled and mds_ssl_enabled }}"
```

please confirm both keystore/truststore needed

---

use vars instead of hardcoding